.handwritten_files

```diff
@@ -18,12 +18,10 @@ _config.yml
 .gitignore
 .handwritten_files
 .smithyrs-githash
```

Comment:
this is autogenerated too right?

Reply:
good catch! I wasn't committing this for anything but the last commit and I really should be doing it for every commit, let me fix that.